cmd/snap/cmd_userd_test.go

```diff
@@ -69,16 +69,15 @@ func (s *userdSuite) TestUserd(c *C) {
 		}()
 
 		needle := "io.snapcraft.Launcher"
-		for i := 0; i < 10; i++ {
+		for i := 0; i < 100; i++ {
```

Comment:
The total wait of this loop is now one second - this may be an issue on slow VMs (think autopkgtest). Not sure if it is, just saysing that this changes the total wait of this loop.

Reply:
1 second seemed plenty to me, but I can kick it to 10 seconds.